src/components/util/layout.js

```diff
@@ -224,7 +224,7 @@ export const textContentWidth = memoize(function(
 ){
     var contElem = document.createElement(containerElementType);
     contElem.className = "off-screen " + (containerClassName || '');
-    contElem.innerHTML = textContent;
+    contElem.innerHTML = textContent + contElem.offsetHeight;
```

Comment:
Not sure this line is needed (I think it adds the value of height, like "138" or something, to end of textcontent, for purpose of the calculation), but yeah probably doesn't hurt to add some extra chars/padding to end. (Maybe could just use a comment that is happening; not too important though).

Reply:
@alexkb0009 Yes, I realized that the line was really unnecessary. I just asked @ugur-camoglu  whether there is a recent change since when we were testing, it was something different. (%99 sure)